src/com/company/Main.java

```diff
@@ -6,11 +6,15 @@ public static void main(String[] args) {
 	// write your code here
         System.out.println("Our team");
         System.out.println("11 members");
+        printMyName();
         printDiana();
     }
 
     public static void printDiana() {
         System.out.println("Diana");
     }
 
+    public static void printMyName(){
```

Comment:
Please use a suggestive name for the method

Reply:
done